includes/User.php

```diff
@@ -2685,6 +2685,14 @@ public function isAnon() {
 	}
 
 	/**
+	 * Whether this user is Wikia staff or not
+	 * @return bool
+	 */
+	public function isStaff() {
+		return in_array( 'staff', $this->getEffectiveGroups() );
```

Comment:
I was not suggesting replacing all the instances, but just starting the pattern. I think we eventually need to do that for all such user groups. But, sure, it is not for this PR :)